src/main/java/com/example/gradecalculator/enums/UserNames.java

```diff
@@ -2,9 +2,7 @@
 
 public enum UserNames {
     TRAINEE("Trainee"),
-    TRAINER("Trainer"),
-    STUDENT("Student");
-
+    TRAINER("Trainer");
```

Comment:
this does not have to be touched in this ticket?

Reply:
it wouldn't be used in the future I removed it, when we found out that there's no need to use GradeSystem

Reply:
okay, shall revert to previous state?

Reply:
yes. bonus: create an issue in the project to think about what to do with it